ocs_ci/deployment/rosa.py

```diff
@@ -59,7 +59,7 @@ def deploy(self, log_level=""):
 
         """
         if (
-            config.ENV_DATA["appliance_mode"]
+            config.ENV_DATA.get("appliance_mode", "")
```

Comment:
It is more like cosmetic change rather than functional, but the `ENV_DATA["appliance_mode"]` is defined and used as boolean, so I would suggest to not set default value to empty string but to bollean:
```suggestion
            config.ENV_DATA.get("appliance_mode", False)
```
or without any custom default parameter, which means, that it will return `None`:
```suggestion
            config.ENV_DATA.get("appliance_mode")
```

Reply:
updated